media-libs/vulkan-layers/vulkan-layers-9999.ebuild

```diff
@@ -46,9 +47,9 @@ multilib_src_configure() {
 		-DBUILD_WSI_XCB_SUPPORT=$(usex X)
 		-DBUILD_WSI_XLIB_SUPPORT=$(usex X)
 		-DBUILD_TESTS=OFF
-		-DGLSLANG_INSTALL_DIR="${EPREFIX}/usr"
-		-DCMAKE_INSTALL_INCLUDEDIR="${EPREFIX}/usr/include/vulkan/"
-		-DSPIRV_HEADERS_INSTALL_DIR="${EPREFIX}/usr/include/spirv"
+		-DGLSLANG_INSTALL_DIR="${ESYSROOT}/usr"
```

Comment:
This doesn’t seem right for any of these cases if the variables are genuinely used for installation — we never write to ESYSROOT, we just find headers there.

May need to check if the build system is abusing these variables to find stuff OR if these changes were just added with other changes and there was confusion about which had the needed effect.

Given this package is only going to install one thing, maybe the other variables are for finding? “spirv is installed here”, not “install spirv here”?

Reply:
I think these variables are for informing cmake where to find various dependencies, not for specifying locations to install files from this project. I'll double check.

Reply:
Yes, I believe this is correct:

* `GLSLANG_INSTALL_DIR`: https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/master/CMakeLists.txt#L272
* `SPIRV_HEADERS_INSTALL_DIR`: https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/master/CMakeLists.txt#L268

I'm unsure about `CMAKE_INSTALL_INCLUDEDIR` though.

Reply:
Okay, fantastic, I've checked and I agree with those two. CMAKE_INSTALL_INCLUDEDIR seems like it should genuinely be ```${EPREFIX}``` though as it's only used for installing headers to the system:
```
install(FILES ${LAYER_UTIL_FILES} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/vulkan)
```

Reply:
I've rechecked all these variables, and they're fine now